test_date_to_day.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import date_to_day
+from datetime import datetime
+import logging
+
+logger = date_to_day.setup_logging()
+logger.disabled = True
+
```

Comment:
he tests cover various input scenarios for gather_input, which is great. However, the calculate_day_of_week function is only tested with valid datetime objects, and there's a test with a string that raises an AttributeError.\

```
def test_calculate_day_of_week_specific_date():
    date = datetime(2024, 8, 21)  # Known Wednesday
    result = date_to_day.calculate_day_of_week(date)
    assert result == "Wednesday"

```

Reply:
It sounds like you want to test calculate_day_of_week() with an invalid date object - let me know if I'm interpreting your comment incorrectly. If that's the case, I don't think it's possible to create an invalid date object. datetime.strptime() and datetime() throw an error if the date is invalid.